assets/javascripts/discourse/templates/components/fields/da-period-field.hbs

```diff
@@ -0,0 +1,28 @@
+<div class="field control-group">
+  {{fields/da-field-label label=label field=field}}
+
+  <div class="controls">
+    {{i18n "discourse_automation.triggerables.recurring.every"}}
+
+    {{input
+      type="number"
+      class="input-small"
```

Comment:
Would prefer more descriptive names than input-small and medium, what is small or medium could change based on context.